Formula/ruby@1.8.7.rb

```diff
@@ -0,0 +1,205 @@
+class RubyAT187 < Formula
```

Comment:
From a previous comment https://github.com/Homebrew/homebrew-core/pull/8767#issuecomment-272139269

That should be the equivalent of an ABI version.  Turns out on previous comment I was wrong for ruby 1.8 and we could use `ruby@1.8` and `class RubyAT18` indeed.